scans/auth/plans_and_scripts/webappsec/notes.txt

```diff
@@ -0,0 +1 @@
+Failing due to SSL/TLS SSL_ERROR_UNSUPPORTED_VERSION during login. If you force Firefox to accept TLS 1.1 then it works but you encounter an expired (2022) cert (as of 2025-03-06).
```

Comment:
We should test with a browser proxying through ZAP not directly.

Reply:
But whatever if we'd rather leave it failing with no explanation I'm fine with that too, just go ahead and close this.

Reply:
Or should we just remove it from the tests?

Reply:
I'm fine if we remove it (but it would still be interesting to know if it worked once TLS is tweaked).

Reply:
I can't select less than 1.2 in ZAP, likely due to Java restrictions.

Reply:
It's not worth fighting with. I'll just close this.